### PR TITLE
Git branch name generator extension

### PR DESCRIPTION
**Pull request by michaelbonner**
## Description

New extension `git-branch-name-generator`. Generate a branch name like: `chore/npm-updates`, `bug/fix-the-menu`, or `feature/new-extension` using a dropdown and a text field. The text will get converted to kebab-case.

## Screencast

https://user-images.githubusercontent.com/1122945/211171807-f70356cd-6543-418f-a5b5-55258083eda7.mov

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/git-branch-name-generator/package.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  "author": "mbonner",
+  "categories": [
+    "Developer Tools"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Generate Git Branch Name",
+      "subtitle": "From type and string inputs",
```
Comment (pernielsentikaer) [inline extensions/git-branch-name-generator/package.json:16]:
```suggestion
      "subtitle": "",
```
Doesn't add any contextual value

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension and I have some feedback ready for you:

* Consider using [useForm](https://developers.raycast.com/utilities/react-hooks/useform?q=useForm) - this also supports validation of empty fields
* Consider either [resetting the form](https://developers.raycast.com/api-reference/user-interface/form#imperative-api) or popToRoot after [submitting](https://developers.raycast.com/api-reference/window-and-search-bar#poptoroot)  

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Comment by michaelbonner** [comment]
Awesome! Thank you so much for the review @pernielsentikaer  

I added `useForm`. Super easy to implement 👏
I also now use `popToRoot`. Way better user experience now

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**